src/csharp/Microsoft.Spark/Sql/PicklingUdfWrapper.cs

```diff
@@ -35,16 +42,19 @@ internal object Execute(int splitIndex, object[] input, int[] argOffsets)
     [UdfWrapper]
     internal class PicklingUdfWrapper<T, TResult>
     {
+        [NonSerialized]
+        private readonly bool?[] _sameT = new bool?[1];
```

Comment:
No need to create an array?

Reply:
all values in the array will have default values.  The default value of a `Nullable` is `null`

Reply:
Oh I meant that this has only one element, so no need to create an array.

Reply:
ah got it.  Changed it to use a non array for the 1 param case.